spec/benchmark/memory/report_spec.rb

```diff
@@ -20,12 +20,11 @@
   end
 
   def create_measurement
-    metrics = {
```

Comment:
With Ruby 2.7 this is not the same, a Hash passed in with keys isn't named parameters.

Reply:
I'm cool with this, though we can also express this as `Measurement.new(**metrics)`

Reply:
I can finish this PR once CI is running.